site/content/docs/5.2/layout/containers.md

```diff
@@ -24,7 +24,7 @@ See them in action and compare them in our [Grid example]({{< docsref "/examples
 Please note that we apply an extra-padding on our containers (see [our mixins](#mixins))
 
 {{< bs-table "table" >}}
```

Comment:
Some values are not correct in this table.

Tested quickly with:
```html
<div class="container bg-dark" style="height: 40px">container</div>
<div class="container-sm bg-warning" style="height: 40px">container-sm</div>
<div class="container-md bg-warning" style="height: 40px">container-md</div>
<div class="container-lg bg-warning" style="height: 40px">container-lg</div>
<div class="container-xl bg-warning" style="height: 40px">container-xl</div>
<div class="container-xxl bg-warning" style="height: 40px">container-xxl</div>
<div class="container-fluid bg-warning" style="height: 40px">container-fluid</div>
```

Please check if the behavior is correct. If so, update the values in this table to reflect it.

Reply:
I think that they are great 🤔